src/assets/source/styles/modules/_mixins.scss

```diff
@@ -14,7 +14,7 @@
   &::-moz-placeholder {
     color: #{$color};
   }
-  &::-ms-placeholder {
```

Comment:
The right pseudo class is `-ms-input-placeholder` that was added afterwards. `-ms-placeholder` does not exist (Yeah, MS does not use the same name scheme as the others :smiley:)